eng/containers/UbuntuNetCore2Keyring/ci.yml

```diff
@@ -0,0 +1,26 @@
+trigger:
+- master
+
+pool:
+  vmImage: 'ubuntu-18.04'
+
+variables:
+  containerRegistry: 'azsdkengsys'
+  imageRepository: 'dotnet/ubuntu_netcore2_keyring'
```

Comment:
This is a great start, thanks for boot strapping the process. We will likely want to make this a little more general so it can build and publish all DockerFiles in the container directory. While we can start with just the one can you please go ahead and move the ci.yml file into the containers folder instead so we can easily expand it when we add more containers?

Reply:
Ok